usaspending_api/common/etl/spark.py

```diff
@@ -261,3 +261,47 @@ def merge_delta_table(spark: SparkSession, source_df: DataFrame, delta_table_nam
         WHEN NOT MATCHED THEN INSERT *
     """
     )
+
+
+def diff(left: DataFrame, right: DataFrame, unique_key_col="id", compare_cols=[], collect=False):
+    """
+    Compares two dataframes that share a schema and returns difference at a row level. 
+    Meant to be run in a databricks notebook.
+    Args:
+        left: assumes a DataFrame wtih a schema that matches the right DataFrame
+        right: assumes a DataFrame wtih a schema that matches the right DataFrame
+        unique_key_col: indidcates a unique indentification column for comparison
+        compare_cols: list of columns to be compared, must exist in both left and right dataframe
+        collect: spark function to limit number of rows returned
```

Comment:
While collect is a function on its own, here the parameter is a boolean flag indicating whether to collect the results and return those or not. Should read something more like:
```
collect (bool): flag indicating whether to simply show and print the results (if False) or to instead run the Spark ``.collect()`` function on the diff results DataFrame and return that list of ``Row`` objects
```

Reply:
That clarifies that arg, I'll use similar language.